core/views/site.py

```diff
@@ -118,15 +117,15 @@ def search_view(request):
 
 @login_required
 def search_user_json(request):
-    result = {"users": search_user(request.GET.get("query", ""), True)}
+    result = {"users": search_user(request.GET.get("query", ""))}
     return JsonResponse(result)
 
 
 @login_required
 def search_json(request):
     result = {
-        "users": search_user(request.GET.get("query", ""), True),
-        "clubs": search_club(request.GET.get("query", ""), True),
+        "users": search_user(request.GET.get("query", "")),
```

Comment:
```suggestion
        "users": search_user(request.GET.get("query", ""), as_json=True),
```
Ça c'est si je me trompe pas et que le True c'était bien pour l'argument as_json

Reply:
J'ai fait dans l'autre sens, j'ai retiré le paramètre `as_json` dans la fonction. Il était juste pas pris en compte dedans.